app/forms/steps/capital/property_form.rb

```diff
@@ -37,17 +37,19 @@ def persist!
       end
 
       def before_save
+        record.address = nil if is_home_address&.yes?
+        record.property_owners.destroy_all if has_other_owners.no?
         self.custom_house_type = nil unless custom_house_type?
       end
 
       def person_has_home_address?
         crime_application.applicant.home_address?
       end
 
-      # TODO: use proper partner policy once we have one.
-      def include_partner?
-        YesNoAnswer.new(crime_application.client_has_partner.to_s).yes?
-      end
+      # # TODO: use proper partner policy once we have one.
+      # def include_partner?
+      #   YesNoAnswer.new(crime_application.client_has_partner.to_s).yes?
+      # end
```

Comment:
Did you mean to delete this? I think Tim has added a shared module `OwnershipConfirmation` that you might want to use. Example: https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/blob/main/app/forms/steps/capital/investments_form.rb#L4